core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -206,11 +210,25 @@ object ConsoleProducer {
       .describedAs("size")
       .ofType(classOf[java.lang.Integer])
       .defaultsTo(1024*100)
-    val propertyOpt = parser.accepts("property", "A mechanism to pass user-defined properties in the form key=value to the message reader. " +
-      "This allows custom configuration for a user-defined message reader. Default properties include:\n" +
-      "\tparse.key=true|false\n" +
-      "\tkey.separator=<key.separator>\n" +
-      "\tignore.error=true|false")
+    val propertyOpt = parser.accepts("property",
+      """A mechanism to pass user-defined properties in the form key=value to the message reader. This allows custom configuration for a user-defined message reader.
+        |Default properties include:
+        | parse.key=false
+        | parse.headers=false
+        | ignore.error=false
+        | key.separator=\t
+        | headers.delimiter=\t
+        | headers.separator=,
+        | headers.key.separator=:
+        |Default parsing pattern when:
+        | parse.headers=true & parse.key=true:
```

Comment:
Should we use `and` rather than `&` ?

Reply:
Now i remember the reason why I chose &
![snap](https://user-images.githubusercontent.com/17885538/145125783-ae0e418a-cf2f-4b83-8ba8-d83df3fb9b9e.png)

The lines break in an unfortunate way if and is used. What do you think?

Reply:
Maybe it's time to use more than 80 columns! But that's a discussion for another KIP.

As all these `parse.*` default to `false`, what about only listing the configs set to `true` in the example?
For example:
`parse.headers=false & parse.key=true` -> `parse.key=true`